e2e/journey-suites/src/choice-confirm-poll.test.ts

```diff
@@ -11,7 +11,7 @@ import { username, password } from './utils/demo-user.js';
 
 test('Test happy paths on test page', async ({ page }) => {
   const { clickButton, navigate } = asyncEvents(page);
-  await navigate('/?journey=TEST_LoginWithMiscCallbacks');
+  await navigate('/?tree=TEST_LoginWithMiscCallbacks&clientId=tenant');
```

Comment:
Is there a reason we changed all of these over to `tree` from `journey`? I'd like to retire the use of "tree".